tests/FakeItEasy.IntegrationTests/FakingClassesTests.cs

```diff
@@ -12,10 +11,7 @@ public class FakingClassesTests
         [Test]
         public void Should_be_able_to_get_a_fake_value_of_uri_type()
         {
-            using (Fake.CreateScope(new NullFakeObjectContainer()))
-            {
-                A.Fake<Uri>();
-            }
+            A.Fake<Uri>();

```

Comment:
> "because we can create fakes of type 'System.Uri'"

Although it doesn't make a lot of sense, since these fakes won't be configurable and they won't represent valid URIs :wink: (not that it really matters for the purpose of this test)